stixcore/util/scripts/end2end_testing.py

```diff
@@ -0,0 +1,150 @@
+import os
+import sys
+import shutil
+from pathlib import Path
+from xml.etree import ElementTree as Et
+
+import pytest
+
+from stixcore.data.test import test_data
+from stixcore.ephemeris.manager import Spice, SpiceKernelManager
+from stixcore.io.soc.manager import SOCManager
+from stixcore.processing.L0toL1 import Level1
+from stixcore.processing.LBtoL0 import Level0
+from stixcore.processing.TMTCtoLB import process_tmtc_to_levelbinary
+from stixcore.products.product import Product
+from stixcore.tmtc.packets import TMTC
+
+pytest.skip
```

Comment:
Is this needed on the CI?

Reply:
removed

Reply:
this is needed to keep a reference to pytest "alive" as isort/flake8 will remove it otherwise
but this script assumes that the test config to be loaded
i added a comment